README.md

```diff
@@ -34,6 +34,7 @@ If you are interested in what drove the need for this checkout [the why section]
    - _suggested_ Set `GCLOUD_PROJECT` environment variable to match the Google Project you would like to use. This needs to be on the process running cypress, so it should be before `cypress open` or `cypress run` in npm scripts. cross-env is a helpful way to do this to support multiple platforms and is how it is done in examples.
    - Pass `projectId` into `cypressFirebasePlugin` options when initializing (see comment in next step)
 1. Generate and download a service account as described in [the firebase-admin setup documentation](https://firebase.google.com/docs/admin/setup#initialize-sdk). Save this to a local file within the project which you confirm is within your `.gitignore` - often `./serviceAccount.json`. Make sure YOU DO NOT COMMIT THIS FILE - it is sensitive and will give others admin access to your project.
+   - **Note**: When running tests locally, ensure the `SERVICE_ACCOUNT` is in your environment variables. I.E. `SERVICE_ACCOUNT=$(cat ./serviceAccount.json) cypress open`
```

Comment:
FWIW if you have a service account file then you can use GOOGLE_APPLICATION_CREDENTIALS env variable to point to the path instead of needing to write the contents to SERVICE_ACCOUNT env var [as described further down in the README](https://github.com/prescottprue/cypress-firebase/blob/b50c49064ba258cdf7d79b122d623efb7008efda/README.md?plain=1#L413) - that said, since you ran into this, I'm sure you aren't alone! I'll merge for now and later maybe try rewording the section to include the multiple options